src/PSPackageProject.psm1

```diff
@@ -708,6 +708,25 @@ function Initialize-PSPackageProject {
         ModuleRoot = $ModuleRoot
     }
 
+    # make pspackageproject.json
+    $jsonPrj =
+    @{
+        SourcePath = "src"
+        ModuleName = "${ModuleName}"
+        TestPath = 'test'
+        HelpPath = 'help'
+        BuildOutputPath = 'out'
+        Culture = [CultureInfo]::CurrentCulture.Name # This needs to be settable
+    } | ConvertTo-Json
+
+    if ($(${PSVersionTable}.PSEdition) -eq 'Desktop') {
```

Comment:
Is there a reason that's not just `$PSVersionTable.PSEdition`? Also I tend to invert this check just in case a lower PowerShell version is targeted (I know we don't currently support that): `$PSVersionTable.PSEdition -eq 'Core'` or even `$PSEdition -eq 'Core'` (`$PSEdition` is an automatic variable by itself)

Reply:
Fixed